README.md

```diff
@@ -193,4 +193,4 @@ I am grateful to all previous project reviewers for their advice. Thanks to them
 
 This project is [MIT](./LICENSE) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make a video presentation and add the link to the README file as stated in the requirement. ℹ️ 

<img width="780" alt="Untitled" src="https://github.com/umair-rao/Budget-app/assets/103386183/5e2cfd54-47b3-463e-92aa-3135bb069d14">

Reply:
Dear reviewer, as per requirement it is **not** compulsory to share only links in **Readme** file, However I have already provided link in **PR description**. I am also attaching link here for your reference. 

Presentation: https://www.loom.com/share/14eec9fdfc9f4b57983cbc2a480e586e?sid=da24f5ca-1efa-4914-b30c-e84fa3b4130d

Reply:
- [ ] @umair-rao Great work on your presentation, however, I only saw your face at the beginning for a few seconds, It is a requirement to show the computer screen and your face simultaneously in a small picture during the presentation.
![image](https://github.com/umair-rao/Budget-app/assets/61408860/bee031e2-cf4c-4e39-91cb-3f15554d8bbe)
nb Although it's not a requirement, it's nice having the presentation link in the readme, this makes it easier to access it by the public.